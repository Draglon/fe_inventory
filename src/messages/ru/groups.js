const groups = {
  "Groups": {
    "layout": {
      "title": "Группы",
      "description": "Группы - описание",
    },
    "title": "Группы",
  },
};

export default groups;