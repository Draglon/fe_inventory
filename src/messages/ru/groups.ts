const groups = {
  "Groups": {
    "layout": {
      "title": "Группы",
      "description": "Группы - описание",
    },
    "title": "Группы",
    "addProduct": "Добавить продукт",
  },
};

export default groups;