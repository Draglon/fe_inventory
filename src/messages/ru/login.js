const login = {
  "Login": {
    "layout": {
      "title": "Авторизоваться",
      "description": "Авторизоваться - описание",
    },
    "title": "Авторизоваться",
  },
};

export default login;