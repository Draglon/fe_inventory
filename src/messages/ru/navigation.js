const navigation = {
  "Navigation": {
    "parishes": "Приходы",
    "groups": "Группы",
    "products": "Продукты",
    "users": "Пользователи",
    "settings": "Настройки"
  },
};

export default navigation;