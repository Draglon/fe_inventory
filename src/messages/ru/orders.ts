const orders = {
  "Orders": {
    "layout": {
      "title": "Приходы",
      "description": "Приходы - описание",
    },
    "title": "Приходы",
  },
};

export default orders;