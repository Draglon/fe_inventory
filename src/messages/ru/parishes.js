const parishes = {
  "Parishes": {
    "layout": {
      "title": "Приходы",
      "description": "Приходы - описание",
    },
    "title": "Приходы",
  },
};

export default parishes;