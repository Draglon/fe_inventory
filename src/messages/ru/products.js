const products = {
  "Products": {
    "layout": {
      "title": "Продукты",
      "description": "Продукты - описание",
    },
    "title": "Продукты",
  },
};

export default products;