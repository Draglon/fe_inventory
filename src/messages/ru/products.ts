const products = {
  "Products": {
    "layout": {
      "title": "Продукты",
      "description": "Продукты - описание",
    },
    "title": "Продукты",
    "filter": {
       "type": {
        "label": "Тип:",
        "option": {
          "all": "Все",
          "phone": "Телефон",
          "monitor": "Монитор"
        }
      },
      "specification": {
        "label": "Спецификация:",
        "option": {
          "all": "Все",
          "phone": "Телефон",
          "monitor": "Монитор"
        }
      },
    },
    "guarantee": {
      "from": "с",
      "to": "по"
    }
  },
};

export default products;