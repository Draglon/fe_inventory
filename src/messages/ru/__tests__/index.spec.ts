import ru from "..";

it("ru locale matches snapshot", () => {
  expect(ru).toMatchSnapshot();
});
