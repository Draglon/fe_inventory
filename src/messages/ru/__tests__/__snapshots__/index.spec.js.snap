// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`ru locale matches snapshot 1`] = `
{
  "Groups": {
    "addProduct": "Добавить продукт",
    "layout": {
      "description": "Группы - описание",
      "title": "Группы",
    },
    "title": "Группы",
  },
  "LocaleSwitcher": {
    "label": "Выбрать язык",
    "locale": "{locale, select, ru {Русский} ua {Украинский} en {Английский} other {Unknown}}",
  },
  "Login": {
    "layout": {
      "description": "Авторизоваться - описание",
      "title": "Авторизоваться",
    },
    "title": "Авторизоваться",
  },
  "Navigation": {
    "groups": "Группы",
    "orders": "Приходы",
    "products": "Продукты",
    "settings": "Настройки",
    "users": "Пользователи",
  },
  "Orders": {
    "layout": {
      "description": "Приходы - описание",
      "title": "Приходы",
    },
    "title": "Приходы",
  },
  "Products": {
    "filter": {
      "specification": {
        "label": "Спецификация:",
        "option": {
          "monitor": "Монитор",
          "phone": "Телефон",
        },
      },
      "type": {
        "label": "Тип:",
        "option": {
          "monitor": "Монитор",
          "phone": "Телефон",
        },
      },
    },
    "guarantee": {
      "from": "с",
      "to": "по",
    },
    "layout": {
      "description": "Продукты - описание",
      "title": "Продукты",
    },
    "title": "Продукты",
  },
  "Registration": {
    "description": "Пожалуйста заполните все поля",
    "form": {
      "email": {
        "errors": {
          "required": "Пожалуйста, введите электронную почту!",
        },
        "label": "Электронная почта",
        "placeholder": "Введите электронную почту",
      },
      "password": {
        "errors": {
          "maxLength": "Длина должна быть не более 20 символов",
          "minLength": "Длина должна быть не менее 6 символов",
          "required": "Пожалуйста, введите пароль!",
        },
        "label": "Пароль",
        "placeholder": "Введите пароль",
      },
      "userName": {
        "errors": {
          "required": "Пожалуйста, введите свое имя пользователя!",
        },
        "label": "Имя пользователя",
        "placeholder": "Введите имя пользователя",
      },
    },
    "layout": {
      "description": "Регистрация - описание",
      "title": "Регистрация",
    },
    "title": "Регистрация",
  },
  "Settings": {
    "layout": {
      "description": "Настройки - описание",
      "title": "Настройки",
    },
    "title": "Настройки",
  },
  "Users": {
    "layout": {
      "description": "Пользователи - описание",
      "title": "Пользователи",
    },
    "title": "Пользователи",
  },
  "shared": {
    "add": "Добавить",
    "cancel": "Отменить",
    "confirm": "Подтвердить",
    "create": "Создать",
    "delete": "Удалить",
    "email": "Електронная почта",
    "logIn": "Авторизоваться",
    "logout": "Выйти",
    "modal": {
      "removeParish": {
        "title": "Вы уверены, что хотите удалить этот приход?",
      },
      "removeProduct": {
        "title": "Вы уверены, что хотите удалить этот продукт?",
      },
    },
    "next": "Следующий",
    "password": "Пароль",
    "previous": "Предыдущий",
    "profile": "Профиль",
    "save": "Сохранять",
    "search": "Поиск",
    "settings": "Настройки",
    "signUp": "Регистрация",
    "status": {
      "free": "В наличии",
      "new": "Новый",
      "repair": "Ремонт",
      "reserved": "Зарезервировано",
      "sold": "Продан",
      "used": "Б/У",
    },
  },
}
`;
